tox.ini

```diff
@@ -17,6 +17,7 @@ set_env =
     PYTHONPATH = {tox_root}/lib:{[vars]src_path}
     PYTHONBREAKPOINT=ipdb.set_trace
     PY_COLORS=1
+    PYTHONDONTWRITEBYTECODE=1
```

Comment:
Why is this needed?

Reply:
To disable creating bytecode during tests.  Having bytecode created during testing leads to issues when debuging